### DISCUSSION TITLE
Shipping methods order conditions - add payment gateway condition

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/commerce#3828] [Ideas] [answered]
It would be very useful if we could allow specific shipping methods only if specific payment gateway is selected.

**Comment by lukeholder** [comment, marked as answer]
This is now possible in Craft Commerce 5.3+

![CleanShot 2025-02-03 at 12 45 03@2x](https://github.com/user-attachments/assets/0c57f8e9-05d8-43d6-9cd8-73c7d29fc250)
